x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/single_bulk_create.ts

```diff
@@ -62,6 +62,10 @@ export interface SingleBulkCreateResponse {
   createdItemsCount: number;
 }
 
+function stringBreviate(text, count, insertDots){
+  return text.slice(0, count) + (((text.length > count) && insertDots) ? "..." : "");
+}
```

Comment:
Rather than use this function you would be better off using the lodash one:
https://lodash.com/docs/4.17.15#truncate

Reply:
How to use this?
![image](https://user-images.githubusercontent.com/44448956/79243604-b47c6980-7e2a-11ea-833f-b4e66e18df2c.png)
the error says that _ is not defined